ruby-training/trainees/alan/test_katas/01/math_maddness/spec/math_maddness_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rspec/autorun'
+
+class Aritmetic_operations
+    
+    def self.add(first_number, second_number)
+       first_number +  second_number
+    end
+    
+    def self.substraction(first_number, second_number)
+        first_number - second_number
+    end
+
+    def self.multiply(first_number, second_number)
+        first_number * second_number
+    end
+
+    def self.divide(first_number, second_number)
+        first_number / second_number
+    end
+end
+describe Aritmetic_operations do
+    describe "#add" do
+        it "Sum the two numbers and the result is equal to 16" do
+             calculus = Aritmetic_operations.add(12, 4)
+            expect(calculus).to eq(16)
+        end
+    end
+
+    describe "#substraction" do
+        it "The first number is substracted by the second number, the result is 10" do
```

Comment:
Again, if you look at the console output when running this test, you'll see it doesn't make much sense as a sentence...

![image](https://user-images.githubusercontent.com/5218163/51553508-a498f400-1e38-11e9-9f35-f104f866cb06.png)

It should look like "ArithmeticOperations.subtract returns the result of subtracting two numbers" or something similar.